.github/workflows/ci.yml

```diff
@@ -0,0 +1,189 @@
+---
+name: "CI"
+on:  # yamllint disable
+  - "push"
+  - "pull_request"
+
+jobs:
+  black:
+    runs-on: "ubuntu-20.04"
```

Comment:
It would be interesting to test with `runs-on: "ghcr.io/nautobot/nautobot-dev:latest"`.  This container already has poetry, black and all of the other dependencies...  Same with the flake8 test

Reply:
Hm, you can't specify any docker image under "runs-on":
https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#jobsjob_idruns-on

I would keep it as it is for now...

Reply:
my mistake, I was thinking we could run it on any arbitrary container

Reply:
Yeah, you would probably need to start a service container.